### ISSUE TITLE
Heroku discontinuing free plans

### ISSUE THREAD

**Issue by BraXzy** [syncforynab/fintech-to-ynab#393]
Hey Scott, I just came across this news that Heroku is ending all of their free offerings: https://techcrunch.com/2022/08/25/heroku-announces-plans-to-eliminate-free-plans-blaming-fraud-and-abuse

I'm guessing this impacts anyone who has this hosted there. Is there an alternative offering to host the app you'd recommend? Cheers!

**Comment by scottrobertson** [comment]
Unfortunately I don't have any recommendations for free hosting. There are plenty of ways to do it, but all require a lot more work to do. 

My recommendation if you want it to be as cheap as you can without managing servers etc is to just use YNAB Direct Import, and if you really want webhooks, then use Sync for YNAB.

**Comment by deanlongstaff** [comment]
@scottrobertson Apologies to piggy back off this conversation, do you know if they're any plans to implement webhook syncing directly into the YNAB application? I'm a heavy Monzo user and would love to utilize the webhooks for instant importing, but it would feel cleaner if this was part of the actual YNAB service. I know it's not a user friendly setup process though unless there's a way to automate the creation of webhooks. Cheers 😃

**Comment by scottrobertson** [comment]
> @scottrobertson Apologies to piggy back off this conversation, do you know if they're any plans to implement webhook syncing directly into the YNAB application? I'm a heavy Monzo user and would love to utilize the webhooks for instant importing, but it would feel cleaner if this was part of the actual YNAB service. I know it's not a user friendly setup process though unless there's a way to automate the creation of webhooks. Cheers 😃 

Sadly not. It's too complex for users and it would just create a lot of support load. Sync for YNAB supports them still and makes it a little easier (but at an extra cost).

**Comment by deanlongstaff** [comment]
> > @scottrobertson Apologies to piggy back off this conversation, do you know if they're any plans to implement webhook syncing directly into the YNAB application? I'm a heavy Monzo user and would love to utilize the webhooks for instant importing, but it would feel cleaner if this was part of the actual YNAB service. I know it's not a user friendly setup process though unless there's a way to automate the creation of webhooks. Cheers 😃 
> 
> Sadly not. It's too complex for users and it would just create a lot of support load. Sync for YNAB supports them still and makes it a little easier (but at an extra cost). 

Fair enough. Is sync for ynab able to run in parallel to the built in direct import feature for one account, or does it have to replace it?

The use case for this would be incase some webhooks are missed, direct import would sync back up later and import the missing transactions.


**EDIT: I'm now hosting `fintech-to-ynab` and I can confirm the webhooks work perfectly alongside direct import. When a webhook is received it simply creates a new transaction in my already existing connected account in YNAB 🚀** I am seeing some duplicate transactions created by the normal sync job, however I'm hoping as i categorise and rename payees it will get better at matching them up.
